6-downloading-and-running-existing-tasks/README.md

```diff
@@ -125,7 +125,7 @@ PARAMETERS:
 Let's quickly check on the status of a specific package using `bolt`:
 
 ```
-$ bolt task run package action=status name=bash --nodes node1
```

Comment:
This fails on `node2` and `node3`, because earlier we only installed puppet on `node1`.